Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,797 @@
+- 3xx builds always use 2xx tooling (never the 3xx SDK).
+- 3xx artifacts are only used for distribution, never as build inputs for
+  other bands.
+- Servicing builds use source-built SDKs from the previous releases of the
+  same band (or 2xx for 3xx).
+- For bootstrapping of 2xx/3xx feature bands, stage 2 does not require the 1xx
+  shared components as input. The artifacts produced by stage 1 contain all
+  required packages.
+
+## Build Command Arguments
+
+Feature band builds make use of these relevant command-line arguments:
+
+- `--with-packages` - Provides PSB artifacts
+- `--with-shared-components` - Provides shared component artifacts from 1xx build
+- `--with-sdk` - Specifies the SDK to use for building
+
+## Distro Maintainer Workflows
+
+The following sections describe the workflows for different scenarios.
+
+### Input Artifacts Summary
+
+**For 1xx band builds:**
+- **Bootstrap**:
+    - PSB artifacts: Microsoft-built previous N.0.1xx release
+    - SDK: Microsoft-built previous N.0.1xx release
+- **Servicing**:
+    - PSB artifacts: Source-built previous N.0.1xx release
+    - SDK: Source-built previous N.0.1xx release
+
+**For 2xx band builds:**
+- **Initial release (N.0.200)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Source-built previous N.0.1xx release
+    - SDK: Source-built previous N.0.1xx release
+- **Bootstrap initial release (N.0.200)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Microsoft-built previous N.0.1xx release
+    - SDK: Microsoft-built previous N.0.1xx release
+- **Bootstrap (N.0.201+)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Microsoft-built previous N.0.2xx release
+    - SDK: Microsoft-built previous N.0.2xx release
+- **Servicing (N.0.201+)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Source-built previous N.0.2xx release
+    - SDK: Source-built previous N.0.2xx release
+
+**For 3xx band builds:**
+- **Initial release (N.0.300)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Source-built previous N.0.2xx release
+    - SDK: Source-built previous N.0.2xx release
+- **Bootstrap (N.0.300)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Microsoft-built previous N.0.2xx release
+    - SDK: Microsoft-built previous N.0.2xx release
+- **Bootstrap (N.0.301+)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Microsoft-built previous N.0.2xx release
+    - SDK: Microsoft-built previous N.0.2xx release
+- **Servicing (N.0.301+)**:
+    - Shared component artifacts: Source-built current N.0.1xx release
+    - PSB artifacts: Source-built previous N.0.2xx release
+    - SDK: Source-built previous N.0.2xx release
```

Comment:
@mmitche / @MichaelSimons 

Is this what we agreed on for 3xx? You would never use 3xx SDK _or_ artifacts to build the next version? You would always use 2xx?

Reply:
That is my understanding.

Reply:
Yep.